--- fuse.c	2010-03-20 08:55:12.000000000 +0700
+++ fuse.c~	2010-03-25 07:57:57.000000000 +0700
@@ -126,10 +126,7 @@
 
 } start_files_t;
 
-#ifdef ANDROID
-#else
 static int fuse_init(int argc, char **argv);
-#endif
 
 static int creator_init( void );
 static void fuse_show_copyright(void);
@@ -141,10 +138,7 @@
 				 start_files_t *start_files );
 static int do_start_files( start_files_t *start_files );
 
-#ifdef ANDROID
-#else
 static int fuse_end(void);
-#endif
 
 #ifdef UI_WIN32
 int fuse_main(int argc, char **argv)
@@ -184,11 +178,7 @@
 
 #include "mempool.h"
 
-#ifdef ANDROID
-int fuse_init(int argc, char **argv)
-#else
 static int fuse_init(int argc, char **argv)
-#endif
 {
   int error, first_arg;
   char *start_scaler;
@@ -772,11 +762,7 @@
 }
 
 /* Tidy-up function called at end of emulation */
-#ifdef ANDROID
-fuse_end(void)
-#else
 static int fuse_end(void)
-#endif
 {
   /* Must happen before memory is deallocated as we read the character
      set from memory for the text output */
